site/app/templates/admin/admin_gradeable/AdminGradeableCreate.twig

```diff
@@ -77,15 +77,14 @@
     <div class="option-desc col-md-12">
         {% if action == 'new' or action == 'template' %}
             <!-- Gradeable Type radio buttons (readonly in edit mode) -->
-            <div class="option-title">Gradable Type</div>
-            <div class="option-alt"><a target=_blank href="http://submitty.org/instructor/create_edit_gradeable#types-of-gradeables">What is the type of the gradeable <i style="font-style:normal;" class="helpicon"></i></a></div>    
+            {# (VPAT) fieldset/legend required for gradeable type container radio group #}
+            <fieldset><legend><div class="option-title">Gradeable Type</div></legend>
+            <div class="option-alt"><a target=_blank href="http://submitty.org/instructor/create_edit_gradeable#types-of-gradeables">What is the type of the gradeable <i style="font-style:normal;" class="helpicon"></i></a></div>
             <span id="required_type" >(Required)</span>
         {% else %}
             <b style='font-size:18px'>Gradeable type</b>: <span id="gradeable-type-string">{{ type_string }}</span>
             <br />
         {% endif %}
-    </div>
```

Comment:
These two lines were removed as they were breaking up the `<fieldset>` opened at line 81 (closes at line 127).  This removal also appears to not have changed the UI layout in my testing.